src/test/java/com/feelmycode/parabole/controller/EventControllerTest.java

```diff
@@ -145,24 +154,38 @@ public void createEvent() throws JSONException {
                         .host("parabole.com"),
                     prettyPrint()),
                 preprocessResponse(prettyPrint()),
+                requestFields(
+                    fieldWithPath("userId").type(JsonFieldType.NUMBER).description("사용자 아이디"),
+                    fieldWithPath("createdBy").type(JsonFieldType.STRING).description("이벤트 생성자"),
+                    fieldWithPath("type").type(JsonFieldType.STRING).description("이벤트 유형"),
+                    fieldWithPath("title").type(JsonFieldType.STRING).description("이벤트 제목"),
+                    fieldWithPath("startAt").type(JsonFieldType.STRING).description("이벤트 시작 일시"),
+                    fieldWithPath("endAt").type(JsonFieldType.STRING).description("이벤트 종료 일시"),
```

Comment:
일시는 어떤식으로 전송하는지 전송 형태를 작성해주세요

```suggestion
                    fieldWithPath("startAt").type(JsonFieldType.STRING).description("이벤트 시작 일시 (yyyy-MM-dd'T'HH:mm:ss)"),
                    fieldWithPath("endAt").type(JsonFieldType.STRING).description("이벤트 종료 일시 (yyyy-MM-dd'T'HH:mm:ss)"),
```